### ISSUE TITLE
reranker训练了一段时间就会出现loss的跳跃式增长，然后再也无法下降

### ISSUE THREAD

**Issue by zougengpeng** [FlagOpen/FlagEmbedding#865]
export CUDA_VISIBLE_DEVICES=1,2,3,4,5,6;
nohup torchrun --nproc_per_node 6 -m FlagEmbedding.reranker.run \
--output_dir ./saved_path \
--model_name_or_path ./bge-reranker-v2-m3 \
--train_data ./training_data.jsonl \
--learning_rate 6e-5 \
--fp16 \
--num_train_epochs 20 \
--per_device_train_batch_size 1 \
--gradient_accumulation_steps 4 \
--dataloader_drop_last True \
--train_group_size 4 \
--max_len 1024 \
--weight_decay 0.01 \
--logging_steps 10 \
--log_level error \
--save_steps  100 \
--seed 43

{'loss': 0.2524, 'grad_norm': 13.925312995910645, 'learning_rate': 5.8433823529411766e-05, 'epoch': 0.52}
{'loss': 0.2185, 'grad_norm': 9.776123046875, 'learning_rate': 5.842801857585139e-05, 'epoch': 0.53}
{'loss': 0.2577, 'grad_norm': 12.018136978149414, 'learning_rate': 5.8422213622291024e-05, 'epoch': 0.53}
{'loss': 0.2168, 'grad_norm': 33.0804557800293, 'learning_rate': 5.841640866873065e-05, 'epoch': 0.53}
{'loss': 0.298, 'grad_norm': 32.5804443359375, 'learning_rate': 5.841060371517028e-05, 'epoch': 0.53}
{'loss': 0.3283, 'grad_norm': 13.820213317871094, 'learning_rate': 5.8404798761609906e-05, 'epoch': 0.53}
{'loss': 0.2134, 'grad_norm': 13.766897201538086, 'learning_rate': 5.839899380804954e-05, 'epoch': 0.54}
{'loss': 0.5352, 'grad_norm': 16.255470275878906, 'learning_rate': 5.8393188854489164e-05, 'epoch': 0.54}
{'loss': 0.591, 'grad_norm': 18.531259536743164, 'learning_rate': 5.838738390092879e-05, 'epoch': 0.54}
{'loss': 0.3433, 'grad_norm': 105.62297821044922, 'learning_rate': 5.838157894736843e-05, 'epoch': 0.54}
{'loss': 0.4859, 'grad_norm': 1.4658299684524536, 'learning_rate': 5.8375773993808054e-05, 'epoch': 0.54}
{'loss': 1.2463, 'grad_norm': 4.336298942565918, 'learning_rate': 5.836996904024768e-05, 'epoch': 0.55}
{'loss': 1.1062, 'grad_norm': 59.8530158996582, 'learning_rate': 5.8364164086687304e-05, 'epoch': 0.55}
{'loss': 0.8432, 'grad_norm': 24.684803009033203, 'learning_rate': 5.8358359133126936e-05, 'epoch': 0.55}
{'loss': 1.3545, 'grad_norm': 3.453885555267334, 'learning_rate': 5.835255417956657e-05, 'epoch': 0.55}
{'loss': 0.9285, 'grad_norm': 45.06439971923828, 'learning_rate': 5.8346749226006194e-05, 'epoch': 0.55}
{'loss': 0.7556, 'grad_norm': 27.40998077392578, 'learning_rate': 5.834094427244582e-05, 'epoch': 0.56}
{'loss': 1.3183, 'grad_norm': 1.8719521760940552, 'learning_rate': 5.833513931888545e-05, 'epoch': 0.56}
{'loss': 1.4117, 'grad_norm': 1.9000672101974487, 'learning_rate': 5.832933436532508e-05, 'epoch': 0.56}
{'loss': 1.4152, 'grad_norm': 1.7894184589385986, 'learning_rate': 5.832352941176471e-05, 'epoch': 0.56}
{'loss': 1.4099, 'grad_norm': 1.6880428791046143, 'learning_rate': 5.8317724458204334e-05, 'epoch': 0.56}
{'loss': 1.4018, 'grad_norm': 1.664863109588623, 'learning_rate': 5.8311919504643966e-05, 'epoch': 0.57}
{'loss': 1.398, 'grad_norm': 1.6947050094604492, 'learning_rate': 5.830611455108359e-05, 'epoch': 0.57}
{'loss': 1.3888, 'grad_norm': 1.7623062133789062, 'learning_rate': 5.830030959752322e-05, 'epoch': 0.57}
{'loss': 1.3893, 'grad_norm': 1.7477459907531738, 'learning_rate': 5.829450464396285e-05, 'epoch': 0.57}
{'loss': 1.408, 'grad_norm': 1.6833128929138184, 'learning_rate': 5.828869969040248e-05, 'epoch': 0.57}
{'loss': 1.3883, 'grad_norm': 1.68731689453125, 'learning_rate': 5.828289473684211e-05, 'epoch': 0.57}
{'loss': 1.3825, 'grad_norm': 1.656215786933899, 'learning_rate': 5.827708978328173e-05, 'epoch': 0.58}

跳跃式增长后模型predict出来的 scores全部一致

**Comment by staoxiao** [comment]
You can try to reduce the `lr`, and you need to check the quality of the training data.

**Comment by ChaoyuHuang** [comment]
问下，这个训练预料pos: List[str]只能有一条数据吗？
